custom/icds_reports/models/aggregate.py

```diff
@@ -848,6 +848,10 @@ class DailyAttendance(models.Model, AggregateMixin):
     image_name = models.TextField(null=True)
     pse_conducted = models.SmallIntegerField(null=True)
     state_id = models.TextField(null=True)
+    open_bfast_count = models.IntegerField(null=True)
```

Comment:
Don't we already have these indicators somewhere? Especially PSE, THR, and HCM have to already exist in the SDD?

Reply:
Hey, the PSE, THR, HCM. IN this PR are not the number of days beneficiary got them. These are number of days AWC distributed them